### ISSUE TITLE
Should we record relevant printed or digital material related to a source?

### ISSUE THREAD

**Issue by MikeJ1971** [ilrt/Beyond2022TeiTemplate#15]
We can have bibliographies in the header. Do we want to do that? 

For the MPESE project, we recorded printed versions of pamphlets and made a distinction between the seventeenth-century printed versions of a pamphlet in manuscript form, modern editions, and modern journal articles or books that discussed a particular pamphlet. See the end of [RaleighSpeechDeath1618.pdf](https://github.com/ilrt/Beyond2022TeiTemplate/files/5056001/RaleighSpeechDeath1618.pdf) to see how the bibliographies were used.

In our example, I've created a bibliography that lists the printed volume by Paul and Brendan; and a link to the blog post I wrote for the Jean Golding Institute about the work I did with Brendan on the roll.

```
            <listBibl>
               <bibl type="book">
                  <editor><name><forename>Paul</forename><surname>Dryburgh</surname></name></editor>
                  <editor><name><forename>Brendan</forename><surname>Smith</surname></name></editor>
                  <title>Handbook &amp; select calendar of sources for medieval Ireland in the
                     National Archives of the United Kingdom</title>
                  <publisher>Four Courts Press</publisher>
                  <pubPlace>Dublin</pubPlace>
                  <date when="2015">2005</date>
               </bibl>
               <bibl type="website">
                  <author><name><forename>Mike</forename><surname>Jones</surname></name></author>
                  <title
                     ref="https://jeangoldinginstitute.blogs.bristol.ac.uk/2020/08/10/digital-humanities-meets-medieval-financial-records/"
                     >Digital Humanities meets Medieval Financial Records</title>
                  <publisher>Jean Golding Instute, University of Bristol</publisher>
                  <date when="2020-08-10">10 August 2020</date>
               </bibl>
            </listBibl>
```

We can reduce the markup if needed. For example, we could reduce the editor and author To

```
<editor>Paul Dryburgh</editor>
```

Or reduce the whole bibliographic item:

```
    <bibl>Paul Dryburgh and Brendan Smith. Handbook &amp; select calendar of sources for medieval Ireland in the
                     National Archives of the United Kingdom. (Dublin, 2005)</bibl>
```

However, this reduces options on rendering bibliographies to different citation styles.

**Comment by crooksp** [comment]
Yes, we would like to be able to encode such references or cross-references. We would ideally include those items in the Beyond 2022 database, so they will themselves have a UID that could be encoded here.

**Comment by munnellg** [comment]
For linking to a bibliography, we can either link to sources that exist in the database, or sources that exist outside the database.

Sources outside the database are referenced using a URL, so if it isn't expected that a bibliography item will be in the B2022 database, then the main thing that you want in the listBibl element is a URL such as a DOI which can be used to find the item.

If the source is in the B2022 database then we can use the document's unique reference number to create a link.

**Comment by MikeJ1971** [comment]
Look at how we can link to bibliographic items in the Beyond 2022
